[2022-10-03 20:52:57,863] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_star_wars.trigger_star_wars_agg_dag manual__2022-10-03T20:52:42.654227+00:00 [queued]>
[2022-10-03 20:52:57,878] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ingest_star_wars.trigger_star_wars_agg_dag manual__2022-10-03T20:52:42.654227+00:00 [queued]>
[2022-10-03 20:52:57,879] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-10-03 20:52:57,879] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-10-03 20:52:57,880] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-10-03 20:52:57,902] {taskinstance.py:1259} INFO - Executing <Task(TriggerDagRunOperator): trigger_star_wars_agg_dag> on 2022-10-03 20:52:42.654227+00:00
[2022-10-03 20:52:57,910] {standard_task_runner.py:52} INFO - Started process 2350 to run task
[2022-10-03 20:52:57,914] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ingest_star_wars', 'trigger_star_wars_agg_dag', 'manual__2022-10-03T20:52:42.654227+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/ingest_starwars_data.py', '--cfg-path', '/tmp/tmpq9un51cs', '--error-file', '/tmp/tmpqyanbus3']
[2022-10-03 20:52:57,918] {standard_task_runner.py:77} INFO - Job 19: Subtask trigger_star_wars_agg_dag
[2022-10-03 20:52:58,005] {logging_mixin.py:109} INFO - Running <TaskInstance: ingest_star_wars.trigger_star_wars_agg_dag manual__2022-10-03T20:52:42.654227+00:00 [running]> on host 86c7e9112604
[2022-10-03 20:52:58,095] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=xxxx@xxxxx.com
AIRFLOW_CTX_DAG_OWNER=Ram
AIRFLOW_CTX_DAG_ID=ingest_star_wars
AIRFLOW_CTX_TASK_ID=trigger_star_wars_agg_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-10-03T20:52:42.654227+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-03T20:52:42.654227+00:00
[2022-10-03 20:52:59,580] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/api/common/experimental/trigger_dag.py:91: DeprecationWarning: Calling `DAG.create_dagrun()` without an explicit data interval is deprecated
  dag_hash=dag_bag.dags_hash.get(dag_id),

[2022-10-03 20:52:59,610] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ingest_star_wars, task_id=trigger_star_wars_agg_dag, execution_date=20221003T205242, start_date=20221003T205257, end_date=20221003T205259
[2022-10-03 20:52:59,651] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-10-03 20:52:59,678] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
